[2024-06-25T09:43:55.372+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T09:43:55.528+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-25T09:06:00+00:00 [queued]>
[2024-06-25T09:43:55.549+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-25T09:06:00+00:00 [queued]>
[2024-06-25T09:43:55.550+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 6
[2024-06-25T09:43:55.617+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-25 09:06:00+00:00
[2024-06-25T09:43:55.634+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=261) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T09:43:55.636+0000] {standard_task_runner.py:63} INFO - Started process 269 to run task
[2024-06-25T09:43:55.646+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_apis_final', 'check_previous_run_status', 'scheduled__2024-06-25T09:06:00+00:00', '--job-id', '3716', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting_apis.py', '--cfg-path', '/tmp/tmp12fngofx']
[2024-06-25T09:43:55.648+0000] {standard_task_runner.py:91} INFO - Job 3716: Subtask check_previous_run_status
[2024-06-25T09:43:55.771+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_apis_final.check_previous_run_status scheduled__2024-06-25T09:06:00+00:00 [running]> on host ced2311e92d2
[2024-06-25T09:43:55.978+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_apis_final' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-25T09:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-25T09:06:00+00:00'
[2024-06-25T09:43:55.979+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T09:43:56.033+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-25T09:43:56.034+0000] {python.py:237} INFO - Done. Returned value was: start_run
[2024-06-25T09:43:56.035+0000] {branch.py:36} INFO - Branch into start_run
[2024-06-25T09:43:56.035+0000] {skipmixin.py:178} INFO - Following branch start_run
[2024-06-25T09:43:56.102+0000] {skipmixin.py:246} INFO - Skipping tasks [('skip_run', -1)]
[2024-06-25T09:43:56.176+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T09:43:56.213+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_apis_final, task_id=check_previous_run_status, run_id=scheduled__2024-06-25T09:06:00+00:00, execution_date=20240625T090600, start_date=20240625T094355, end_date=20240625T094356
[2024-06-25T09:43:56.268+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T09:43:56.316+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-25T09:43:56.318+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
